src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -13,23 +20,23 @@ public void countIn(int in) {
     }
 
     public boolean callCheck() {
-        return total == -1;
+        return total == COUNTFINISH;
     }
 
     public boolean isFull() {
-        return total == 11;
+        return total == MAX;
     }
 
     protected int peekaboo() {
         if (callCheck()) {
-            return -1;
+            return COUNTFINISH;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)